### DISCUSSION TITLE
Feature Request: mux-video and media-chrome thumbnails

### DISCUSSION THREAD

**Discussion by bourhaouta** [muxinc/elements#649] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Which Mux Elements/Packages does this apply to? Select all that apply

mux-video

### Description

Heyo! I'm asking if it's possible to set thumbails/storyboard to the `mux-video` component so `media-chrome` can pick them up, I tried to add them as a normal video, but not sure if that's the right way
```html
<mux-video>
...here
</mux-video>
```

### Expected Behavior

Setting `mux-video` thumbnails

**Comment by luwes** [comment, marked as answer]
thanks for the question! yes this works as you expected.
the track elements will be picked up internally by the native video element and that info is passed to Media Chrome.

see an example here
https://media-chrome.mux.dev/examples/vanilla/media-elements/mux-video.html
